sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/auth/Token.java

```diff
@@ -0,0 +1,27 @@
+package com.mesosphere.sdk.dcos.auth;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+import java.util.Date;
+
+/**
+ * Token represents a JWT authentication token issued by DC/OS Bouncer IAM service.
+ */
+public class Token {
```

Comment:
I was only trying to make package boundary and not leak underlying library into dcos auth implementation. I can easily remove `Token` class and use `DecodedJWT` instead.